### PR TITLE
Match func_i1_80192518

### PR DESCRIPTION
**Pull request by Vulto**

### CODE REVIEW

**Review by sonicdcer — APPROVED** [review]

File: src/overlays/ovl_i1/sf_ve1.c [hunk 1]
```diff
@@ -154,17 +333,31 @@ void func_i1_80192CB0(Actor* actor) {
...
-#pragma GLOBAL_ASM("asm/us/nonmatchings/overlays/ovl_i1/sf_ve1/func_i1_801933B4.s")
+void func_i1_801933B4(Actor* arg0) {
+    arg0->scale = -1.0f;
+    arg0->obj.pos.y += 488.0f;
+}
 
 #pragma GLOBAL_ASM("asm/us/nonmatchings/overlays/ovl_i1/sf_ve1/func_i1_801933DC.s")
 
-#pragma GLOBAL_ASM("asm/us/nonmatchings/overlays/ovl_i1/sf_ve1/func_i1_801934D0.s")
+void func_i1_801934D0(Actor* arg0) {
```
Comment (sonicdcer) [inline src/overlays/ovl_i1/sf_ve1.c:348]:
```suggestion
void func_i1_801934D0(Actor* actor) {

File: src/overlays/ovl_i1/sf_ve1.c [hunk 2]
```diff
@@ -154,17 +333,31 @@ void func_i1_80192CB0(Actor* actor) {
...
+    Matrix_Translate(gGfxMatrix, 0.0f, -488.0f, 0.0f, 1U);
+    Animation_DrawSkeleton(0, &D_900D164, arg0->vwork, NULL, NULL, arg0, &gIdentityMatrix);
+}
 
-#pragma GLOBAL_ASM("asm/us/nonmatchings/overlays/ovl_i1/sf_ve1/func_i1_80193540.s")
+void func_i1_80193540(Object_80* arg0) {
+    f32 temp_fv1;
+
+    temp_fv1 = arg0->obj.pos.z;
```
Comment (sonicdcer) [inline src/overlays/ovl_i1/sf_ve1.c:356]: temp_fv1 looks fake, can you try removing it?

File: src/overlays/ovl_i1/sf_ve1.c [hunk 3]
```diff
@@ -178,9 +371,19 @@ void func_i1_80192CB0(Actor* actor) {
 
 #pragma GLOBAL_ASM("asm/us/nonmatchings/overlays/ovl_i1/sf_ve1/func_i1_80198414.s")
 
-#pragma GLOBAL_ASM("asm/us/nonmatchings/overlays/ovl_i1/sf_ve1/func_i1_80198594.s")
+void func_i1_80198594(Boss* arg0) {
```
Comment (sonicdcer) [inline src/overlays/ovl_i1/sf_ve1.c:374]:
```suggestion
void func_i1_80198594(Boss* boss) {

File: src/overlays/ovl_i1/sf_ve1.c [hunk 4]
```diff
@@ -178,9 +371,19 @@ void func_i1_80192CB0(Actor* actor) {
...
+    arg0->obj.pos.z = gPlayer->pos.z;
+    if ((arg0->timer_052 <= 0) && (gPlayer->state_1C8 == PLAYERSTATE_1C8_3)) {
+        func_i1_80198414();
+    }
+}
 
-#pragma GLOBAL_ASM("asm/us/nonmatchings/overlays/ovl_i1/sf_ve1/func_i1_801985E4.s")
+void func_i1_801985E4(Boss* arg0) {
```
Comment (sonicdcer) [inline src/overlays/ovl_i1/sf_ve1.c:381]:
```suggestion
void func_i1_801985E4(Boss* boss) {

File: src/overlays/ovl_i1/sf_ve1.c [hunk 5]
```diff
@@ -154,17 +333,31 @@ void func_i1_80192CB0(Actor* actor) {
 
 #pragma GLOBAL_ASM("asm/us/nonmatchings/overlays/ovl_i1/sf_ve1/func_i1_80192E2C.s")
 
-#pragma GLOBAL_ASM("asm/us/nonmatchings/overlays/ovl_i1/sf_ve1/func_i1_80192EA4.s")
+void func_i1_80192EA4(Actor* arg0) {
```
Comment (sonicdcer) [inline src/overlays/ovl_i1/sf_ve1.c:336]:
```suggestion
void func_i1_80192EA4(Actor* actor) {

**Review by sonicdcer — CHANGES_REQUESTED** [review]

**Comment by Vulto** [comment]
One of these functions are not matching anymore. 
I'll have to check one by one now. 
I never should've added these functions in this PR.

**Comment by sonicdcer** [comment]
> One of these functions are not matching anymore. I'll have to check one by one now. I never should've added these functions in this PR.

image
There's no need to check them one by one. They are non matching because you applied the suggestions here, which you should have done manually on the repo like i suggested in your previous PR. Also, some members have new names in master so you need to accommodate your PR so functions match again.

**Comment by Vulto** [comment]
Yeah i tried that before posting. I t does not worked.
Even after return all functions of this PR to asm, compilation stills not matching.
I think i'll be easier if i just delete my fork.

**Comment by sonicdcer** [comment]
> Yeah i tried that before posting. I t does not worked. Even after return all functions of this PR to asm, compilation stills not matching. I think i'll be easier if i just delete my fork.

I'm sorry that you're finding this frustrating. Let me get home, I'll PR a fix to your branch, which will fix all your problems so we can merge your PR.

**Comment by Vulto** [comment]
I appreciate it. That's kind of you.

**Comment by sonicdcer** [comment]
There, i made a PR to this branch in your fork, merge the PR so this is fixed. While you're there, you might want to check out what i did so you can understand why your branch was broken in the first place.

File: src/overlays/ovl_i1/sf_ve1.c (hunk 2 above)
Comment (Vulto) [reply src/overlays/ovl_i1/sf_ve1.c]: Yeah, it's fake. removed on this commit "e31fd0a"

**Review by Vulto — COMMENTED** [review]
Requests addressed

**Pull request merged**